### PR TITLE
Updates for Managed Delivery documentation

### PR DESCRIPTION
**Pull request by luispollo**
This PR includes the following:
- Add the [premonition](https://github.com/lazee/premonition) Jekyll plugin which provides nicer block style elements (notes, tips, warnings, etc.)
- Add a Managed Delivery section to the Concepts page (linked from “How it works” on the homepage), embed @emjburns's and @robfletcher’s talk at Spinnaker Summit
- In the Guides tab, under How-to Guides, add a link to the MD guide which is in the same area
- In the Guides tab, under Service Overviews, add a link to the Keel README
- In the Reference tab, under Architecture, add Keel to the list/diagram of services
- In Guides > User How-tos > Managed Delivery, remove Alpha banner and other language indicating MD is not production ready.

### CODE REVIEW

File: _sass/premonition.scss [hunk 1]
```diff
@@ -0,0 +1,159 @@
+$default-color: #5bc0de;
```
Comment (luispollo) [inline _sass/premonition.scss:1]: This file is auto-generated by the premonition plugin with default styles.

File: concepts/index.md [hunk 2]
```diff
@@ -120,3 +122,27 @@ A *Task* in Spinnaker is an automatic function to perform.
...
+desired state. 
+
+For more details on the inspiration and guiding principles behind
+Managed Delivery, check out [our blog](https://blog.spinnaker.io/managed-delivery-evolving-continuous-delivery-at-netflix-eb74877fb33c),
+or the talk below from Spinnaker Summit 2019.
+
+<iframe src="https://www.youtube.com/embed/mEgvOfmLnlY" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+> info ""
+> _To get started with Managed Delivery, head on over to the [user guide](/guides/user/managed-delivery/)._
```
Comment (luispollo) [inline concepts/index.md:147]: This renders like so:
image

File: reference/architecture/index.md [hunk 3]
```diff
@@ -92,6 +94,14 @@ service.
...
  gate --> igor(Igor);
  igor(Igor) --> echo;
+ 
+ keel(Keel) --> clouddriver;
+ keel --> orca;
+ keel --> front50;
+ keel --> fiat;
+ keel --> echo;
+ keel --> igor;
+ gate --> keel;
```
Comment (luispollo) [inline reference/architecture/index.md:104]: This makes the diagram render like so:
image

It's not super pretty, but I found it hard to control the layout.

File: Gemfile.lock [hunk 4]
```diff
@@ -1,3 +1,11 @@
+GIT
```
Comment (luispollo) [inline Gemfile.lock:1]: Had to update the lock file after adding the `premonition` gem.

**Review by gal-yardeni — APPROVED** [review]
LGTM!

**Comment by luispollo** [comment]
The HTML check errors in the build seem unrelated with my changes. Not sure what to do about them. Please let me know if  you've ever seen those before.

**Review by lorin — APPROVED** [review]
LGTM

Note that I tried to follow the "Local preview" commands in the readme to preview the generated content using Docker, but I couldn't get it to serve content at localhost:4000. I assume these difficulties are unrelated to the changes.

**Pull request merged**